src/index.html

```diff
@@ -8,6 +8,38 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card_container" data-qa="card">
```

Comment:
try to use one BEM block `card` and other children as these block elements
```suggestion
    <div class="card" data-qa="card">
```